dll/win32/shell32/shelldesktop/CChangeNotifyServer.h

```diff
@@ -89,3 +89,29 @@ typedef struct HANDBAG
 #define HANDBAG_MAGIC 0xFACEB00C
 
 HRESULT CChangeNotifyServer_CreateInstance(REFIID riid, void **ppv);
+
+typedef CWinTraits <
+    WS_POPUP | WS_CLIPSIBLINGS | WS_CLIPCHILDREN,
+    WS_EX_TOOLWINDOW
+> CWorkerTraits;
+
+DECLSPEC_SELECTANY ATL::CWndClassInfo g_wcWorker =
+{
+    { sizeof(WNDCLASSEX), 0, CWindowImplBaseT<CWindow, CWorkerTraits>::StartWindowProc,
+      0, 0, NULL, NULL, NULL, (HBRUSH)(COLOR_3DFACE + 1), NULL, _T("WorkerW"), NULL },
+    NULL, NULL, IDC_ARROW, TRUE, 0, _T("")
+};
+
+template <typename T_BASE>
+class CWorker :
+    public CWindowImpl<T_BASE, CWindow, CWorkerTraits>
+{
+public:
+    CWorker() { }
+    virtual ~CWorker() { }
+
+    static ATL::CWndClassInfo& GetWndClassInfo()
```

Comment:
Why not use `DECLARE_WND_CLASS_EX` ?

Reply:
It splits the class definition. I want to unify the classes.

Reply:
I don't understand how it splits the definition.

Reply:
If it duplicates the class definition, then the `RegisterClassEx` will fail and `m_atom` will be zero.

Reply:
Like this:
![VirtualBox_ReactOS_24_01_2022_19_04_34](https://user-images.githubusercontent.com/2107452/150762519-a7f7f938-af8e-4d90-a026-2235551cc01c.png)

Reply:
Thanks for the explanation

Reply:
We can still use `SHCreateWorkerWindowW` instead...

Reply:
Used `SHCreateWorkerWindowW`.